docs/ref/contrib/admin/javascript.txt

```diff
@@ -0,0 +1,74 @@
+
+========================
+Admin javascript tweaks
+========================
+
+.. versionadded:: developement 
+
+Inline hooks
+------------
+
+Sometimes you need to execute some JavaScript or jQuery upon adding or removing inlines in the admin change_form.
```

Comment:
chop "or jQuery" (that's a subset of JavaScript)
Please wrap lines at 79 characters and use backticks around code-type things like variable names, etc.